test/integration/sessions/sessions.spec.prose.test.ts

```diff
@@ -46,6 +46,6 @@ describe('ServerSession', () => {
     expect(events).to.have.lengthOf(operations.length);
 
     // This is a guarantee in node, unless you are performing a transaction (which is not being done in this test)
-    expect(new Set(events.map(ev => ev.command.lsid.id.toString('hex')))).to.have.lengthOf(1);
+    expect(new Set(events.map(ev => ev.command.lsid.id.toString('hex')))).to.have.lengthOf(2);
```

Comment:
why did this change?

Reply:
> Our minimum session usage for all the parallel operations we run in these tests is now 2. With async await we will never put the first created session back before the next operation attempts to pull one out of the pool. Unlike multithreaded drivers that may have some level of nondeterminism, connection checkIn -> release is deterministically after the next operation reaches the checkOut -> acquire step. As we further refactor to async await I don't expect this to go any higher than 2 but it is possible it could decrease again to 1 if the order of async steps allows us to return the session before the next operation tries to acquire one.